content/resources/crownlabs_ssh/crownlabs_ssh.md

```diff
@@ -0,0 +1,94 @@
++++
+fragment = "content"
+#disabled = true
+date = "2021-11-12"
+weight = 100
+# title = "Connecting to CrownLabs with SSH"
++++
+
+## Introduction
+
+**CrownLabs** allows users to access their VMs through a local SSH client (i.e., executed on their local machines).
+
+In this guide you are going to learn how to generate a SSH key pair on your operating systems (Windows, Linux, MacOS), how to load it on **CrownLabs** and access your **CrownLabs**' VM using SSH from a local terminal.
```

Comment:
```suggestion
This guide shows how to generate a SSH key pair on your operating systems (Windows, Linux, MacOS), how to load it on **CrownLabs** and access your **CrownLabs**' VM using SSH from a local terminal.
```

Note: keep the text as short as possible.

Reply:
I changed it.